app/assets/stylesheets/components/_card.scss

```diff
@@ -247,3 +251,98 @@ img.avatar-large {
   position: absolute;
   left: 10px;
 }
+
+.team-cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 16px;
+  padding: 16px;
+  justify-items: center;
+  a {
+    text-decoration: none;
+  }
+}
+
+@media (max-width: 700px) {
+  .team-cards {
+    grid-template-columns: 1fr;
+  }
+}
+
+@media (min-width: 700px) and (max-width: 992px) {
+  .team-cards {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media (min-width: 992px) and (max-width: 1200px) {
+  .team-cards {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+.team-card {
+  width: 250px;
+  height: 300px;
+  text-align: center;
+  border-radius: 16px;
+  background-color: rgba(4, 5, 45, 1);
+  border: 1px solid rgba(4, 5, 45, 1);
+  box-shadow: inset 0px 0px 84px black;
+  background-image: url(asset_path("virtuspro_logo.png"));
+  position: relative;
+  background-position: 50% 25%;
+  background-size: 60%;
+  background-repeat: no-repeat;
+  transition: .2s ease;
+  &:hover {
+    background-size: 100%;
+    transition: .2s ease;
+  }
+  .team-name-card {
+    padding-top: 200px;
+    font-size: 24px;
+    color: white;
+  }
+  .team-region-card {
+    color: #067275;
+    font-weight: bolder;
+    font-family: 'Ubuntu Condensed', serif;
```

Comment:
this is likely not a font for others so it will swap to serif. lets update with a google font